scripts/monorepo/buildTo.js

```diff
@@ -0,0 +1,46 @@
+const { spawnSync } = require('child_process');
```

Comment:
This whole script would be nice to eventually have in just (I'm guessing you eventually plan to replace the rush monorepo template with a lerna one?)

Reply:
yeah i do :)